### ISSUE TITLE
build_all_repos: usage clarification + typos

### ISSUE THREAD

**Issue by jouvin** [quattor/release#73]
I read https://github.com/quattor/release/blob/master/src/scripts/build_all_repos.sh usage and found not very clear what use case this script is addressing. Is it just for testing? Or for building alternative repos to standard ones? Probably a clarification about when to use this script may help.

Also, I found a few typos:

- https://github.com/quattor/release/blob/master/src/scripts/build_all_repos.sh#L47: point to remote upstream -> remote upstream is configured to refer to them.
- https://github.com/quattor/release/blob/master/src/scripts/build_all_repos.sh#L51: CAPN -> CPAN
- https://github.com/quattor/release/blob/master/src/scripts/build_all_repos.sh#L67: I'd suggest
```
RELEASE: if set to 1, it will remove the existing repositories and do some other intrusive cleanups. DON'T USE IT if you are not sure you need it.
```
- https://github.com/quattor/release/blob/master/src/scripts/build_all_repos.sh#L69: clarifiy use case

**Comment by stdweird** [comment]
it is intended to bootstrap and test quattor with minimal dependencies (e.g. a minimal VM with EPEL repo enabled is enough). 

i'll try to make it a bit more clear.

**Comment by stdweird** [comment]
@jouvin should be fixed in #74

**Comment by jrha** [comment]
Fixed by #74
